pyproject.toml

```diff
@@ -111,7 +111,7 @@ test-docs = [
     "groq<0.14.0,>=0.4.2",
     "phonenumbers<9.0.0,>=8.13.33",
     "cohere<6.0.0,>=5.1.8",
-    "mistralai<2.0.0,>=1.0.3",
+    "mistralai<2.0.0,>=1.5.1",
```

Comment:
Mistral doesn't release changelings, so I just updated to the latest version just to be safe